composer/datasets/c4_hparams.py

```diff
@@ -36,8 +35,12 @@ class StreamingC4Hparams(DatasetHparams):
         group_method (str): How to group text samples into token samples. Currently only `truncate` is supported.
         mlm (bool): Whether or not to use masked language modeling. Default: ``False``.
         mlm_probability (float): If ``mlm==True``, the probability that tokens are masked. Default: ``0.15``.
+        shuffle (bool): Whether to shuffle the samples in the dataset. Currently, shards are assigned and consumed with
+            deterministic per-device shard order, but shuffling affects the order of samples via (per-device) shuffle
+            buffers. Default: ``False``.
```

Comment:
This needs to be reworded as StreamingDataset is not deterministic yet. Could you ask @knighton for the best way to reword this description here?

Reply:
"Whether to iterate over the samples in randomized order"?